platform/core/src/DICOMSR/dataExchange.js

```diff
@@ -63,36 +63,35 @@ const downloadReport = (measurementData, additionalFindingTypes = []) => {
  *
  * @param {object[]} measurementData An array of measurements from the measurements service
  * that you wish to serialize.
- * @param {string[]} additionalFindingTypes toolTypes that should be stored with labels as Findings
+ * @param {string[]} options.additionalFindingTypes toolTypes that should be stored with labels as Findings
  */
-const generateReport = (measurementData, additionalFindingTypes) => {
-  const filteredToolState = _getFilteredCornerstoneToolState(
-    measurementData,
-    additionalFindingTypes
-  );
+const generateReport = (measurementData, options = {}) => {
+  const ids = measurementData.map(md => md.id);
+  const filteredToolState = _getFilteredCornerstoneToolState(ids);
```

Comment:
This does not work. Not sure why this was changed

Reply:
We need to add the SeriesDescription through the options when we download locally also right?

I think this additionalFindingTypes thing broke when you rebased on v2-main, I'll update it.

Reply:
Yeah when I did this commit on your branch this additional findings stuff wasn't there. Weird. I've fixed this and pushed.